### PR TITLE
[Footer] create footer | [Carrusel] create carrusel 

### PR DESCRIPTION
**Pull request by agustinhummel**
COMO: usuario
QUIERO: mostrar el footer
PARA: mostrar info de la pagina 
FOOTER

### CODE REVIEW

**Review by lucaaramos — APPROVED** [review]

**Review by Ariel-Alegre — APPROVED** [review]

File: client/src/App.js [hunk 1]
```diff
@@ -9,7 +10,10 @@ function App() {
     <BrowserRouter>
       <Switch>
         <Route exact path='/register'>
-<Register />
+          <Register />
+        </Route>
+        <Route exact path="/">
```
Comment (gabicasas01) [inline client/src/App.js:15]: me quedan dudas, con el exact path se sigue renderizando en todas las rutas de la app?

**Review by gabicasas01 — APPROVED** [review]

File: client/src/index.css [hunk 2]
```diff
@@ -26,10 +26,10 @@ a:hover {
 
 body {
   margin: 0;
-  display: flex;
+  /* display: flex; */
```
Comment (miguelbel00) [inline client/src/index.css:29]: delete this comments

File: client/src/index.css [hunk 3]
```diff
@@ -26,10 +26,10 @@ a:hover {
...
   place-items: center;
-  min-width: 320px;
-  min-height: 100vh;
+  /* width: 100%;
+  min-height: 100vh; */
```
Comment (miguelbel00) [inline client/src/index.css:32]: its necessary this comments?

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
maybe change the name of folder style for "styles" and componentes for "components"
and maybe delete app.css because no one should be work there

File: client/src/index.css (hunk 3 above)
Comment (agustinhummel) [reply client/src/index.css]: ok ya solucinado

**Review by miguelbel00 — APPROVED** [review]
All ok

**Review by iSensative — APPROVED** [review]
Ok

**Pull request merged**